cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -65,8 +65,12 @@ spec:
             mountPath: /var
       volumes:
       - name: influxdb-persistent-storage
+{% if grains['cloud'] == 'gce' %}
```

Comment:
Well there's [influxdb-pv.yaml](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-monitoring/influxdb/influxdb-pv.yaml) in the same directory which does contain templates which are removed with sed in gce provider scripts. The better way would be an emptyDir PV but that isn't supported as of now, see #17354